Amend API method template to describe `DOMException` usage (#15221)
mdn/content: files/en-us/mdn/structures/page_types/api_method_subpage_template/index.md

- Exception2
  - : etc.

> Note that we have two kinds of exceptions: {{domxref("DOMException")}} objects and regular JavaScript exceptions, like {{jsxref("TypeError")}} and {{jsxref("RangeError")}}. A web developer needs to know which object is thrown, and for exceptions that are `DOMException` objects, he also needs to know their `name`. Here is an example where a method can raise a `DOMException` with a name of `IndexSizeError`, a `DOMException` with a name of `SyntaxError` and a JavaScript exception of type `TypeError`:

Reviewer: ```suggestion
Note that we have two kinds of exceptions: {{domxref("DOMException")}} objects and regular JavaScript exceptions, like {{jsxref("TypeError")}} and {{jsxref("RangeError")}}. A web developer needs to know:

 - which object is thrown
 - for exceptions that are `DOMException` objects, the `name` of the exception
 
Here is an example where a method can raise a `DOMException` with a name of `IndexSizeError`, a `DOMException` with a name of `SyntaxError` and a JavaScript exception of type `TypeError`:
```

Author: I try to avoid he, but I missed this one.
